src/main/java/com/sngular/kloadgen/sampler/SamplerUtil.java

```diff
@@ -318,9 +322,12 @@ public static BaseLoadGenerator configureValueGenerator(final Properties props)
     final BaseLoadGenerator generator;
 
     final String valueNameStrategy = jMeterVariables.get(ProducerKeysHelper.VALUE_NAME_STRATEGY);
-
     if (Objects.isNull(valueNameStrategy)) {
-      props.put(ProducerKeysHelper.VALUE_NAME_STRATEGY, ProducerKeysHelper.TOPIC_NAME_STRATEGY);
+      if (SchemaRegistryKeyHelper.SCHEMA_REGISTRY_APICURIO.equalsIgnoreCase(jMeterVariables.get(SchemaRegistryKeyHelper.SCHEMA_REGISTRY_NAME))) {
```

Comment:
```
if (SchemaRegistryKeyHelper.SCHEMA_REGISTRY_APICURIO.equalsIgnoreCase(jMeterVariables.get(SchemaRegistryKeyHelper.SCHEMA_REGISTRY_NAME)) || SchemaRegistryKeyHelper.SCHEMA_REGISTRY_CONFLUENT.equalsIgnoreCase(jMeterVariables.get(SchemaRegistryKeyHelper.SCHEMA_REGISTRY_NAME))) {
        props.put(ProducerKeysHelper.VALUE_NAME_STRATEGY, ProducerKeysHelper.TOPIC_NAME_STRATEGY_CONFLUENT);
      }
```

Reply:
If block statement should not apply when first condition is true

Reply:
Sorry, it seems thar I haven't understood the code;
I think this is the correct one:
```
if (Objects.isNull(valueNameStrategy)) {
  props.put(ProducerKeysHelper.VALUE_NAME_STRATEGY, SchemaRegistryKeyHelper.SCHEMA_REGISTRY_APICURIO.equalsIgnoreCase(jMeterVariables.get(SchemaRegistryKeyHelper.SCHEMA_REGISTRY_NAME)) ? ProducerKeysHelper.TOPIC_NAME_STRATEGY_APICURIO : ProducerKeysHelper.TOPIC_NAME_STRATEGY_CONFLUENT);
} else {
    props.put(ProducerKeysHelper.VALUE_NAME_STRATEGY, valueNameStrategy);
}
```
Change an if statemen for a ternari operator.